specification/eventgrid/data-plane/Microsoft.EventGrid/stable/2024-01-01/SystemEvents.json

```diff
@@ -8425,6 +8425,14 @@
           "type": "string",
           "description": "The type of blob."
         },
+        "accessTier": {
+          "type": "string",
```

Comment:
Should these be extensible enums? I would expect there are well-known values for the access tier. Here is an example of a property that is an extensible enum - https://github.com/Azure/azure-rest-api-specs/pull/29443/files#diff-3aacd38599ce1798dcd5926c41f3866fa8e2411e4186cdf6d9ca0749ba8c42fbR8132

Reply:
The team that publishes events for HNS accounts and FNS accounts are different, and it is difficult to maintain compatibility between the different tiers that are supported for different types of accounts (based on implementation and bandwidth). It might be easier to retain these as strings.

Reply:
Extensible enums are still strings. They just provide a list of well-known values that will be generated on the client. There is no impact on the wire format, but it makes it easier for customers to work with these models.